cmd/api/main.go

```diff
@@ -216,18 +205,8 @@ func createChainIDStack(
 	}
 
 	scAddress := common.HexToAddress(config.Registry.ContractAddress)
-	registry, err := ethereum.NewClient(
-		conn,
-		config.ChainID,
-		scAddress,
-		wallet,
-		tracker,
-	)
-	if err != nil {
-		return chains.ChainStack{}, fmt.Errorf("failed to create ethereum client: %s", err)
-	}
 
-	acl := impl.NewACL(systemStore, registry)
```

Comment:
I realized that this acl component was not using the `registry`, so I removed it